src/core/InterleavedBuffer.js

```diff
@@ -92,6 +94,14 @@ InterleavedBuffer.prototype = {
 
 		return new this.constructor().copy( this );
 
+	},
+
+	onUpload: function ( callback ) {
```

Comment:
I should have put more context. I didn't know what it was until I went to test in the editor. Thanks for reviewing.